charts/cass-operator/templates/configmap.yaml

```diff
@@ -29,15 +28,20 @@ data:
     images:
       system-logger: "k8ssandra/system-logger:v1.10.0"
       config-builder: "datastax/cass-config-builder:1.0.4-ubi7"
+{{- if .Values.image.repositoryOverride }}
```

Comment:
Can you explain `respostioryOverride`? I see the example you added in values.yaml, but I don't understand.

Reply:
The kubernetes clusters I want to use the operator in do not have direct internet access, so I have to poll the images through a proxy repo (nexus). For this I need the possibility to specify another image repository.